lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -118,15 +180,14 @@ class window.Turbolinks
     triggerEvent 'page:receive'
     options.updatePushState ?= true
 
-    if doc = processResponse(xhr, options.partialReplace)
-      reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
-      reflectRedirectedUrl(xhr) if options.updatePushState
+    doc = processResponse(xhr, options.partialReplace)
+    reflectNewUrl url if options.updatePushState
+    nodes = changePage(extractTitleAndBody(doc)..., options)
+    reflectRedirectedUrl(xhr) if options.updatePushState
+
+    updateHead(doc).then ->
```

Comment:
This is happening too late.  At this point, `changePage` will have:
- Already blown away the old document and displayed the new content
- Executed new inline scripts, which'll probably puke because they're referencing unloaded classes